src/psyclone/dynamo0p3.py

```diff
@@ -8193,6 +8247,32 @@ def mesh(self):
         '''
         return self._mesh_properties
 
+    @property
+    def all_updates_are_writes(self):
+        '''
+        :returns: true if all of the arguments updated by this kernel have \
+                  GH_WRITE access, false otherwise.
```

Comment:
I would suggest single quotes around `'GH_WRITE'`. I would also suggest "True" and "False" (start with upper case) for consistency with other docstrings.

Reply:
done